build/Dockerfile

```diff
@@ -47,5 +47,6 @@ ENV GOFLAGS="-buildvcs=false"
 # Since the user does not match the owners of the repo "git rev-parse --is-inside-work-tree" fails and goreleaser does not populate projectName
 # https://stackoverflow.com/questions/72978485/git-submodule-update-failed-with-fatal-detected-dubious-ownership-in-repositor
 RUN git config --global --add safe.directory '*'
+COPY ./sign.sh ./sign_deb.exp ./sign_rpm.exp ./sign_tar.exp /usr/local/bin
```

Comment:
We are only copying the files since they are executed already in the makefile? https://github.com/newrelic/nri-redis/pull/212/files#diff-553b9a740f0151f2e959512198dddbdd0d69c398d29583ca0a4e011d5598932e

Reply:
Yes, copying so that we can use same files for other integrations.